src/main/java/de/bonndan/nivio/assessment/Assessment.java

```diff
@@ -0,0 +1,25 @@
+package de.bonndan.nivio.assessment;
+
+import de.bonndan.nivio.assessment.kpi.KPI;
+
+import java.util.List;
+import java.util.Map;
+
+public class Assessment {
```

Comment:
A short description of the class would be helpful.

Reply:
done